nemo/collections/llm/recipes/gemma2_27b.py

```diff
@@ -220,6 +220,7 @@ def finetune_recipe(
         recipe.optim.config.lr = 5e-6
         recipe.trainer.strategy.tensor_model_parallel_size = 8
         recipe.trainer.strategy.pipeline_model_parallel_size = 2
+        recipe.trainer.strategy.pipeline_dtype = torch.bfloat16
```

Comment:
I think this affects all recipes with PP and should be fixed in the default: https://github.com/NVIDIA/NeMo/pull/11273/files#diff-77182f4b6d0cd76301f8626afb363ed986bd9c6a5574d2622f0825a2569cb31eR86